src/views/admin-quiz-add.phtml

```diff
@@ -38,7 +38,7 @@ require_once('partials/header.phtml');
             <label for="quizQuestions">Questions</label>
             <select name="questions[]" class="form-control" id="quizQuestions" multiple>
                 <?php /** @var $questions \QuizApp\Entity\QuestionTemplate[] */?>
-                <?php foreach ($questions as $question) :?>
+                <?php foreach ($questions->getItems() as $question) :?>
```

Comment:
It would help to add a /** @var ... */ for $question as well inside the foreach so that reader and IDE knows what type of objects you iterate.